Library/Homebrew/test/support/fixtures/cask/everything-systemsettings-caveats.json

```diff
@@ -93,6 +93,7 @@
     "en",
     "eo"
   ],
+  "ruby_source_path": "Formula/everything.rb",
```

Comment:
`brew tests` failing on my Ventura machine from `./test/cask/cask_spec.rb:231` because the actually output is `"ruby_source_path": "Casks/everything.rb"`

Reply:
Weird. Then there also has to be a formula test that is missing this.

Which test is failing, specifically?

Reply:
This is the only failure for me: https://github.com/Homebrew/brew/blob/c51687009428dd061a2dd90f46c11e0c7c1f9212/Library/Homebrew/test/cask/cask_spec.rb#L225-L236

Reply:
Are you on Ventura?

Reply:
Yep.

Reply:
Yeah, this test shouldn't depend on the macOS version and stub it instead.